src/js/components/hero-container.jsx

```diff
@@ -7,8 +7,14 @@ class HeroContainer extends React.Component {
     render () {
 
         return (
-            <div className="container hero-container">
-                <h2>This is the hero container</h2>
+            <div className="hero-container">
+                <h1 className="hero-title">A Y O O O</h1>
```

Comment:
Small suggestion: here we could use the css `letter-spacing` property to achieve the wider spacing between each character, rather than using actual spaces :smile: